src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/integration/PrisonOffenderEventListenerTest.kt

```diff
@@ -54,30 +54,29 @@ class PrisonOffenderEventListenerTest : SqsIntegrationTestBase() {
     "classpath:test_data/transfer-prison.sql",
   )
   fun `prisoner received event should update offender prison`() {
-    assertThat(offenderRepository.findByPrisonNumber(PRISON_NUMBER)?.prisonId).isEqualTo(OLD_PRISON_CODE)
-
-    publishDomainEventMessage(
-      AdditionalInformationTransfer(
-        nomsNumber = PRISON_NUMBER,
-        reason = "TRANSFERRED",
-        prisonId = NEW_PRISON_CODE,
-      ),
+    // Then
```

Comment:
isn't this technically part of the when?

Reply:
yes messed it up again, I never do this why am I doing this now!

Reply:
Should have been Given as its the prep before the code is actioned